Makefile

```diff
@@ -305,7 +316,7 @@ daemon-protobuf: src/newrelic/infinite_tracing/com_newrelic_trace_v1/v1.pb.go
 src/newrelic/infinite_tracing/com_newrelic_trace_v1/v1.pb.go: protocol/infinite_tracing/v1.proto
 	$(MAKE) vendor # Only build vendor stuff if v1.proto has changed. Otherwise
 	               # this rule will be triggered every time the daemon is built.
-	$(VENDOR_BASE)/local/bin/protoc \
+	$(VENDOR_PREFIX)/bin/protoc \
```

Comment:
Should this be `PROTOBUF_C_PREFIX`?

Reply:
I was thinking about it but decided this will require a separate PR for the following reasons:
1. `protoc` is not `protoc-c`. `protoc` is protobuf compiler with support for many languages but C. `protoc` is provided by `protobuf`. protoc-c is built on top of `protoc` and adds support for C language. In this particular target `protoc` is needed, not `protoc-c` and this PR only makes updates to provider of `protoc-c`.
2. [src/newrelic/infinite_tracing/com_newrelic_trace_v1/v1.pb.go](https://github.com/newrelic/newrelic-php-agent/blob/dev/src/newrelic/infinite_tracing/com_newrelic_trace_v1/v1.pb.go) is stored in the repository and this target is not usually triggered in day to day builds. `src/newrelic/infinite_tracing/com_newrelic_trace_v1/v1.pb.go` has to be rebuild manually after `protocol/infinite_tracing/v1.proto` gets an update and the result of the manual build has to be checked into the repository. If this target needs to be rebuilt, not only `protoc` needs to be available, but also Go protocol buffer plugin needs to be manually installed - see [Makefile#L295-L298](https://github.com/newrelic/newrelic-php-agent/blob/dev/Makefile#L295-L298).

Given 1. and especially 2., which requires additional manual effort to properly configure build environment, I decided for this PR not to touch this area of the build system and have it use `protoc` from vendor subdirectory, which will be built by the command on line 317.